### DISCUSSION TITLE
Long (or impossible?) translation

### DISCUSSION THREAD

**Discussion by sfederici** [leopard-js/leopard-mentors#9] [Cleanup Crew]
### Scratch Project URL

https://scratch.mit.edu/projects/556991097/

### What kind of error(s) are you experiencing?

- [X] Leopard is failing to generate JavaScript code
- [ ] The JavaScript code generated by Leopard doesn't work correctly
- [ ] The JavaScript code generated by Leopard is confusing or needs improvement
- [ ] Other

### Description of issue(s)

I don't know if the project is too large for Leopard (29 sprites, 620 scripts) or if there are limitations. After 15 minutes it is still converting.

**Comment by PullJosh** [comment]
I've updated the Leopard website so that it can now process larger projects. So when I try to convert it now, I do **not** get an infinite spinner. (That's an improvement!)

Unfortunately, this project exceeds the 10 MB maximum that we currently enforce:

<img width="732" alt="image" src="https://github.com/user-attachments/assets/e047e4c9-78cc-4ba9-a08e-20496a13cf26">

In the future it might be reasonable to increase this limit, but I want to keep the rules restrictive for now to make sure the website can support itself at the current scale first.

**Reply by sfederici** [reply]
Ok. Let me know, please, if you increase the limit, so that I can try larger projects.

**Reply by PullJosh** [reply]
In the meantime, you can convert large projects locally using the `sb-edit` CLI: https://github.com/leopard-js/sb-edit?tab=readme-ov-file#convert-sb3-project-to-leopard

**Reply by sfederici** [reply]
Thanks a lot
